04-14 07:38:06 imba: False
04-14 07:38:06 model_name: CORAL
04-14 07:38:06 Domain: CWRU
04-14 07:38:06 source: CWRU_1
04-14 07:38:06 target: CWRU_2
04-14 07:38:06 data_dir: ./datasets
04-14 07:38:06 train_mode: single_source
04-14 07:38:06 cuda_device: 0
04-14 07:38:06 save_dir: ./ckpt/CORAL/single_source
04-14 07:38:06 max_epoch: 30
04-14 07:38:06 batch_size: 32
04-14 07:38:06 num_workers: 0
04-14 07:38:06 signal_size: 2048
04-14 07:38:06 random_state: 75
04-14 07:38:06 normlizetype: mean-std
04-14 07:38:06 opt: sgd
04-14 07:38:06 lr: 0.001
04-14 07:38:06 momentum: 0.9
04-14 07:38:06 betas: (0.9, 0.999)
04-14 07:38:06 weight_decay: 0.0005
04-14 07:38:06 lr_scheduler: stepLR
04-14 07:38:06 gamma: 0.2
04-14 07:38:06 steps: 10
04-14 07:38:06 tradeoff: ['exp', 'exp', 'exp']
04-14 07:38:06 dropout: 0.0
04-14 07:38:06 save: False
04-14 07:38:06 load_path: 
04-14 07:38:06 tsne: False
04-14 07:38:06 save_path: ./ckpt/CORAL/single_source/[CWRU_1]To[CWRU_2]_0414-073806
04-14 07:38:06 Detect 4 classes: ['1normal', 'ball', 'inner', 'outer']
04-14 07:38:07 using 1 / 1 gpus
04-14 07:38:09 Source set CWRU_1 number of samples 767.
04-14 07:38:10 Training set number of samples 611.
04-14 07:38:10 Validation set number of samples 156.
04-14 07:38:10 -----Epoch 1/30-----
04-14 07:38:10 current lr: [0.001]
  0%|          | 0/19 [00:00<?, ?it/s] 16%|#5        | 3/19 [00:00<00:00, 28.95it/s] 42%|####2     | 8/19 [00:00<00:00, 38.33it/s] 68%|######8   | 13/19 [00:00<00:00, 39.56it/s] 95%|#########4| 18/19 [00:00<00:00, 40.85it/s]100%|##########| 19/19 [00:00<00:00, 39.49it/s]
04-14 07:38:10 Train-Loss Source Classifier: 1.2522
04-14 07:38:10 Train-Loss CORAL: 0.0783
04-14 07:38:10 Train-Acc Source Data: 0.4309
  0%|          | 0/4 [00:00<?, ?it/s]  0%|          | 0/4 [00:00<?, ?it/s]
Traceback (most recent call last):
  File "train.py", line 95, in <module>
    trainer.train()
  File "/home/workspace/UDA_Bearing_Fault_Diagnosis/models/CORAL.py", line 123, in train
    new_acc = self.test()
  File "/home/workspace/UDA_Bearing_Fault_Diagnosis/models/CORAL.py", line 150, in test
    acc += utils.get_accuracy(pred, target_labels)
  File "/home/workspace/UDA_Bearing_Fault_Diagnosis/utils.py", line 13, in get_accuracy
    assert preds.shape[0] == targets.shape[0]
AttributeError: 'tuple' object has no attribute 'shape'
